test/dynamo/test_functions.py

```diff
@@ -1124,6 +1124,14 @@ def test_numpy_dtype_argument_to_function(x):
 
         return np.empty_like(x, dtype=np.float64)
 
+    @requires_numpy_pytorch_interop
+    @make_test
+    def test_numpy_linalg(x):
+        import numpy as np
+
+        norm = np.linalg.norm(x.numpy(), axis=0)
+        return torch.from_numpy(norm)
```

Comment:
No need of using the `from_numpy`. I added a patch so that the test without `from_numpy` passes in https://github.com/pytorch/pytorch/pull/105333.